content/_index.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Ceres Miller"
++++
+
+{% header(subheading="Product Designer, Glasgow, UK") %}
+- BA(hons) Product Design, University of Edinburgh
+- Design for adaptation
+- Craft, electronics and digital design
+- CircuitPython, Grasshopper
+{% end %}
+
+{% section(link="@/mindseye.md") %}
+
+MIND'S EYE - A Digital Camera Made to be Modified
+-------------------------------------------------
+
+* * *
+
+A project about adaptable digital electronics. What's inside a black box?
+
+![Thumbnail image: Mind's Eye camera with the top plate taken off, showing tools around it](/img/mindseye-6.jpg)
+
+* * *
+
+{% end %}
+
+{% section() %}
```

Comment:
Zola has this neat concept of "shortcodes" which is how I'm managing to add `<section>` tags in markdown. But, there's a bit of a pain point with it. Because they don't seem to be able to be nested. Basically, I'm wondering if you _either_ want to do away with `<section>` (sad 😢 , less semantic) and we can do some stuff with CSS to make those divisions still appear. 

OR we can just do `<section>`s inside html